source/documentation/cli/migrator.html.md.erb

```diff
@@ -1,5 +1,292 @@
+stylesheets that they depend on using the [`@use` rule][], [`@forward` rule][],
+or [`@import` rule][].
+
+```shellsession
+$ sass-migrator module --verbose style.scss
+Migrating style.scss
+$ sass-migrator module --verbose --migrate-deps style.scss
+Migrating style.scss
+Migrating _theme.scss
+Migrating _fonts.scss
+Migrating _grid.scss
+```
+
+### `--dry-run`
+
+This flag (abbreviated `-n`) tells the migrator not to save any changes to
+disk. It instead prints the list of files that it would have changed. This is
+commonly paired with the [`--verbose` option][] to print the contents of the
+changes that would have been made as well.
+
+[`--verbose` option]: #verbose
+
+```shellsession
+$ sass-migrator module --dry-run --migrate-deps style.scss
+Dry run. Logging migrated files instead of overwriting...
+
+style.scss
+_theme.scss
+_fonts.scss
+_grid.scss
+```
+
+#### `--no-unicode`
+
+This flag tells the Sass migrator only to emit ASCII characters to the terminal
+as part of error messages. By default, or if `--unicode` is passed, the migrator
+will emit non-ASCII characters for these messages. This flag does not affect the
+CSS output.
+
+```shellsession
+$ sass-migrator --no-unicode module style.scss
+line 1, column 9 of style.scss: Error: Could not find Sass file at 'typography'.
+  ,
+1 | @import "typography";
+  |         ^^^^^^^^^^^^
+  '
+Migration failed!
+$ sass-migrator --unicode module style.scss
+line 1, column 9 of style.scss: Error: Could not find Sass file at 'typography'.
+  ╷
+1 │ @import "typography";
+  │         ^^^^^^^^^^^^
+  ╵
+Migration failed!
+```
+
+### `--verbose`
+
+This flag (abbreviated `-v`) tells the migrator to print extra information to
+the console. By default, it just prints the name of files that are changed, but
+when combined with the [`--dry-run` option][] it also prints those files' new
+contents.
+
+[`--dry-run` option]: #dry-run
+
+<%# Indent this because otherwise the table-of-contents generator interprets
+the `===` as a heading. %>
+
+    $ sass-migrator module --verbose --dry-run style.scss
+    Dry run. Logging migrated files instead of overwriting...
+
+    style.scss
+    ================================================================================
+    @use "bootstrap" with (
+      $body-bg: #000,
+      $body-color: #111
+    );
+
+    @include bootstrap.media-breakpoint-up(sm) {
+      .navbar {
+        display: block;
+      }
+    }
+
+    --------------------------------------------------------------------------------
+    $ sass-migrator module --verbose style.scss
+    Migrating style.scss
+
+## Migrations
+
+The migrator currently supports only one migration, but expect more to come as
+the Sass language continues to evolve!
+
+### Module
+
+This migration converts stylesheets that use the old [`@import` rule][] to load
+dependencies so that they use the Sass module system via the [`@use` rule][]
+instead. It doesn't just naïvely change `@import`s to `@use`s—it updates
```

Comment:
Consider splitting this up:

This migration updates stylesheets that load dependencies to use the Sass module system. Any usages of the old [`@import` rule][] get converted to the [`@use` rule][].

Reply:
I'd like to keep this as-is, because I feel like the existing wording makes it clearer that you can run the migrator on stylesheets that mix `@import` and `@use` rules.